.spellcheckerdict.txt

```diff
@@ -75,11 +78,12 @@ ie
 [Ii]nfura
 [Ii]oredis
 ipfs
-IPFSContentProcessor
 IPFS('s)?
+IPFSContentProcessor
```

Comment:
is this sort churn?

Reply:
Hmm not sure - spellchecker needed it for ipfs processor